platform/server/integrations/keycloak/initKeycloak.js

```diff
@@ -33,16 +33,110 @@ const PUBLIC_CLIENT_NAME = 'open-volunteer-platform-public'
 const BEARER_CLIENT_NAME = 'open-volunteer-platform-bearer'
 let PUBLIC_CLIENT
 
+let demoUsers = process.env.USE_DEMO_DATA !== 'true' ? []: [
+  {
+    name: 'msash',
```

Comment:
I think we need some designated admin here. 1 or two would be good and they cannot have actions assigned

Reply:
I was thinking just about that. I'll rename the default `admin:admin` to `ovp-admin:ovp-admin`, WDYT?

Reply:
Done through out.